config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
   resources :developers
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  post :search, to: 'developers#search'
```

Comment:
Why post?
You don't create any data, do you?

Reply:
ah yeah!
In this case, better if use `get` method. @jp-ryuji